_overviews/scala3-migration/incompat-dropped-features.md

```diff
@@ -171,51 +193,66 @@ The Scala 3 compiler produces two error messages:
 
 It suggests to use trait parameters which would give us:
 
-```scala
+{% tabs scala-3-initializer_2 %}
+{% tab 'Scala 3 Only' %}
+~~~ scala
 trait Bar(name: String) {
   val size: Int = name.size
 }
 
 object Foo extends Bar("Foo")
-```
+~~~
+{% endtab %}
+{% endtabs %}
 
 Since trait parameters are not available in Scala 2.13, it does not cross-compile.
 If you need a cross-compiling solution you can use an intermediate class that carries the early initialized `val`s and `var`s as constructor parameters.
 
-```scala
+{% tabs shared-initializer_4 %}
+{% tab 'Scala 2 and 3' %}
+~~~ scala
 abstract class BarEarlyInit(val name: String) extends Bar
 
 object Foo extends BarEarlyInit("Foo")
-```
+~~~ 
 
 In the case of a class, it is also possible to use a secondary constructor with a fixed value, as shown by:
-
-```scala
+~~~ scala
 class Fizz private (val name: String) extends Bar {
   def this() = this("Fizz")
 }
-```
+~~~
+{% endtab %}
+{% endtabs %}
 
 ## Existential Type
 
 Existential type is a [dropped feature]({{ site.scala3ref }}/dropped-features/existential-types.html), which makes the following code invalid.
-
-```scala
-def foo: List[Class[T]] forSome { type T } // Error: Existential types are no longer supported
-```
+  
+{% tabs scala-2-existential_1 %}
+{% tab 'Scala 2 Only' %}
+~~~ scala
+def foo: List[Class[T]] forSome { type T } // In Scala 3, Error: Existential types are no longer supported
+~~~
+{% endtab %}
+{% endtabs %}
 
 > Existential type is an experimental feature in Scala 2.13 that must be enabled explicitly either by importing `import scala.language.existentials` or by setting the `-language:existentials` compiler flag.
 
-The proposed solution is to introduce an enclosing type that carries the dependent type:
+In Scala 3, the proposed solution is to introduce an enclosing type that carries the dependent type:
 
-```scala
+{% tabs scala-3-existential_1 %}
+{% tab 'Scala 3 Only' %}
```

Comment:
Actually, this solution cross-compiles with both Scala 2 and 3, so we should say “Scala 2 and 3” here.